### PR TITLE
Update SearchForm

### PR DESCRIPTION
**Pull request by tractorcow**
CMS PR for https://github.com/silverstripe/silverstripe-framework/pull/6692

### CODE REVIEW

**Comment by tractorcow** [comment]
Ok fixed.

**Comment by dhensby** [comment]
Quite a few CI errors @tractorcow :/

**Comment by tractorcow** [comment]
Hm, closer. Let me try again.

**Comment by tractorcow** [comment]
It looks like the default argument to `$sortBy` to postgres searchEngine() method is incorrect. Can work-around it in CMS module for now. https://github.com/silverstripe/silverstripe-cms/pull/1758/commits/5808165e0ca7e6c8f8cbba9645a1018a2fed2cf8

**Comment by tractorcow** [comment]
Oh right, I forgot I'd changed the search form api slightly; I removed direct access to request variables, so you need to assign a proper Request object for testing.

**Comment by tractorcow** [comment]
Cool, postgres fixed.

**Comment by tractorcow** [comment]
Small fix for behat tests at https://github.com/silverstripe/silverstripe-framework/pull/6697

**Comment by tractorcow** [comment]
I've determined that the code coverage failure is due to phpunit 4.8 incompatibilities with php7. Not going to address it this PR.
